ppa/editorial/models.py

```diff
@@ -83,15 +81,34 @@ def route(self, request, path_components):
             return super().route(request, path_components)
 
 
+@register_snippet
+class Person(models.Model):
```

Comment:
I see why you added these under `editorial`, but we might want to use them for the contributor page at some point, so I think they ought to be under `pages`.  What do you think? 

(I had that in mind, but I didn't tell you that or document anywhere - I apologize).

Reply:
Please add a brief docstring documenting what this is and where it's currently used / why it's here.